core/src/main/java/org/infinispan/util/logging/events/EventLogCategory.java

```diff
@@ -0,0 +1,14 @@
+package org.infinispan.util.logging.events;
+
+/**
+ * EventLogCategory.
+ *
+ * @author Tristan Tarrant
+ * @since 8.2
+ */
+public enum EventLogCategory {
+   LIFECYCLE,
```

Comment:
@tristantarrant why not leave some guidelines when each category is supposed to be used?